pinot-core/src/main/java/com/linkedin/pinot/core/io/writer/impl/v1/FixedByteChunkSingleValueWriter.java

```diff
@@ -68,37 +68,44 @@ public FixedByteChunkSingleValueWriter(File file, ChunkCompressor compressor, in
 
   @Override
   public void setInt(int row, int value) {
```

Comment:
Yeah, chunk based writers with compression do support writes at random offsets. Updated javadoc with the same.